man/systemctl.xml

```diff
@@ -1660,8 +1660,7 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
           completion is implemented for property names.</para>
 
           <para>For the manager itself,
-          <command>systemctl show</command> will show all available
-          properties. Those properties are documented in
+          <command>systemctl show</command>will show properties that mostly are derived, or closely match, the options listed in
```

Comment:
maybe: "will show all available properties, most of which are derived, or closely match, ..." - ie, avoid hinting that there might be more properties not shown here